src/wasm/wasm-validator.cpp

```diff
@@ -1490,19 +1490,21 @@ void FunctionValidator::visitReturn(Return* curr) {
 }
 
 void FunctionValidator::visitHost(Host* curr) {
+  shouldBeTrue(
+    getModule()->memory.exists, curr, "Memory operations require a memory");
```

Comment:
We've been still printing and parsing `memory.grow` as `grow_memory` so far but we only updated a test in `test/spec/memory.wast` to use `memory.grow`. But the error was masked because the test there was
```wast
(assert_invalid                                                                  
  (module (func (drop (memory.grow (i32.const 0)))))                             
  "unknown memory"                                                               
)
```
This failed because of parsing error (the parser didn't recognized `memory.grow`) but it was supposed to fail because there's no memory. After we fixed the parser to recognize `memory.grow`, this started to succeed because we didn't have this `shouldBeTrue` line. So added it.